packages/lesswrong/components/posts/PostsItem.jsx

```diff
@@ -179,9 +206,12 @@ class PostsItem extends PureComponent {
       }
 
       return (
-        <div>
-          <CommentIcon className="posts-item-comment-icon" style={commentCountIconStyle}/>
-          <div className="posts-item-comment-count">
+        <div onClick={this.toggleNewComments}
+          className={classNames("posts-item-comments", {"selected":this.state.showNewComments, "highlight-selected":this.state.showHighlight
```

Comment:
Can you extract all the state variables from state at the beginning of the render function? I expect this will make this line a lot easier to read.